Cargo.toml

```diff
@@ -24,10 +24,13 @@ maintenance = { status = "passively-maintained" }
 bindgen = { version = "^0.63", optional = true }
 cc = "^1.0.3"
 pkg-config = "^0.3.3"
+nix = "^0.26.2"
+num_cpus = "^1.15.0"
 
 [lib]
 crate-type = ["lib", "staticlib"]
 
 [features]
 novendor = []
 static = []
+static_dependencies = ["static"]
```

Comment:
I feel like `vendored` is more commonly used (and more succinct) terminology. E.g., [here](https://github.com/sfackler/rust-openssl/blob/a71f492fccaf0743e987cdf12f7032683d951ae9/openssl-sys/Cargo.toml#L18).

Reply:
done